ha/core/system_health/const.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
```

Comment:
Please add copyright and other information in the file header.

Reply:
Added copyright and other information.